crates/subspace-bootstrap-node/src/main.rs

```diff
@@ -192,34 +194,31 @@ async fn main() -> Result<(), Box<dyn Error>> {
             info!("Subspace Bootstrap Node started");
 
             // TODO: spawn this in a dedicated thread
```

Comment:
Why does it need a dedicated thread? There is extremely little work done in there, it should be perfectly fine to run it where it was.

Reply:
Dedicated named threads make it a lot easier to debug any locking/performance issues with the spawned task.

I'm not suggesting we put a lot of work into this. But if a new upstream release makes it easier to run it in a dedicated thread, I think it's a worthwhile change.